docs/developer-experience/artificial-intelligence.md

```diff
@@ -0,0 +1,63 @@
+# Artificial Intelligence
+
+There are a number of AI tools that can improve the developer experience. This article will discuss tooling that is available as well as advice on when it might be appropriate to use such tooling.
+
+## GitHub Copilot
+
+The current version of GitHub Copilot can provide code completion in Visual Studio Code. It is a VS Code extension that can be installed from the VS Code Marketplace. It is currently in beta and requires a GitHub account to use. It is free to use during the beta period. It is currently available for VS Code only, but there are plans to support other IDEs in the future. It is currently available for the following languages: JavaScript, TypeScript, Ruby, Go, Python, C#, C++, and Java. You can find more information about GitHub Copilot [here](https://copilot.github.com/).
+
+Developers on our team have found the following use-cases for GitHub Copilot:
+
+- __Write Documentation__. For example, the above paragraph was written using Copilot.
+- __Write Unit Tests__. Given that setup and assertions are often consistent across unit tests, Copilot tends to be very accurate.
+- __Unblock__. It is often hard start writing when staring at a blank page, Copilot can fill the space with something that may or may not be what you ultimately want to do, but it can help get you in the right headspace.
+
+If you want Copilot to write something useful for you, try writing a comment that describes what your code is going to do - it can often take it from there.
+
+## GitHub Copilot Labs
+
+Copilot has extension that offers additional features that are not yet ready for prime-time. You can install it from the VS Code Marketplace. These features include:
+
+- __Explain__. Copilot can explain what the code is doing in natural language.
+- __Translate__. Copilot can translate code from one language to another.
+- __Brushes__. You can select code that Copilot then modifies inline based on a "brush" you select, for example, to make the code more readable, fix bugs, improve debugging, document, etc.
+- __Generate Tests__. Copilot can generate unit tests for your code. Though currently this is limited to JavaScript and TypeScript.
+
+## GitHub Copilot X
+
+This extention for Copilot is currently in technical preview. It will enable the following additional features:
```

Comment:
If we describe the availability, this page will immediately be outdated when the component is released. To avoid this, change this and say, see ... for availability

Reply:
I think we may just have to update it once it becomes available. How and where to sign-up and how to check whether you are "in" was really hard for folks. I had several people reach out to me for help, so I put the links here.

Reply:
If there is an issue where people can't tell from the product page how to sign up, we should make an effort to connect with the product group to make this easier.

Reply:
As discussed, removing this section. I will consider if this is better for the wiki or just point people to the official documentation.